examples/statistics/errorbar_limits.py

```diff
@@ -1,50 +1,76 @@
 """
-Demo of the errorbar function, including upper and lower limits
+===========================================================
+Demo of how to include upper and lower limits in error bars
+===========================================================
+
+In matplotlib, errors bars can have "limits". Applying limits to the
+error bars essentially makes the error unidirectional. Because of that,
+upper and lower limits can be applied in both the y- and x-directions
+via the `uplims`, `lolims`, `xuplims`, and `xlolims` parameters,
```

Comment:
These should be double backticks because they're not references, but formatting.